ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.15__amd64-nightly-4.15-upgrade-from-stable-4.14.yaml

```diff
@@ -307,8 +307,6 @@ tests:
       BASE_DOMAIN: qe.devcluster.openshift.com
       BASELINE_CAPABILITY_SET: None
       PERSISTENT_MONITORING: "false"
-      UPGRADE_CHECK_RUN_TAGS: not @console and not @storage
-      UPGRADE_PRE_RUN_TAGS: not @console and not @storage
```

Comment:
Shall we also remove the console tag on https://github.com/openshift/release/blob/e8eedfc0df5c362db263ad1ef58ddf65b9fd89fc/ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.15__amd64-nightly-4.15-upgrade-from-stable-4.14.yaml#L151 ?